File: 215 (Kth Largest Element in an Array)/cpp/Solution.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int,vector<int>,greater<int>> mheap;
        for(int i = 0;i<k;i++){
            mheap.push(nums[i]);
        }

        for(int i = k;i<nums.size();i++){
            if (nums[i]>mheap.top()){
                mheap.pop();
                mheap.push(nums[i]);
            }
        }

        return mheap.top();
    }
};

File: 215 (Kth Largest Element in an Array)/java/Solution.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> mheap = new PriorityQueue<>();

        for(int i = 0;i<k;i++){
            mheap.add(nums[i]);
        }

        for(int i = k;i<nums.length;i++){
            if (nums[i]>mheap.peek()){
                mheap.poll();
                mheap.add(nums[i]);
            }
        }
        return mheap.peek();
    }
}